developer_tools/XSDSchemas/Metrics.xsd

```diff
@@ -2,19 +2,29 @@
 <xsd:schema version="1.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema"  elementFormDefault="qualified">
 
     <xsd:complexType name="MetricsData">
+        <xsd:sequence>
+            <xsd:any processContents="skip" minOccurs="0" maxOccurs="unbounded"/>
+        </xsd:sequence>
+        <xsd:attribute name="verbosity" type="verbosityAttr" default="all"/>
+    </xsd:complexType>
+    <!-- as of PR #1573, using subType to delineate the XML options means Metrics cannot be
+    checked using XSD 1.0 -->
```

Comment:
This is just a comment. Basically, XSD 1.0 will not working when we have subType structure. We need to push to use XSD 1.1 instead.